index.tt

```diff
@@ -6,152 +6,251 @@
-      </a>
-      Blog Posts
-    </h2>
-    <ul class="nixos-blogs">
-      [% FOREACH b IN JSON.Escape.json_decode(IO.All.new('blogs.json').all).channel.item.first(5); IF b; %]
-        <li>
-          <div class="nixos-blog">
-            [%
-              USE title = String(HTML.escape(b.title.remove('^[^:]*:').trim));
-              title = title.replace('^[a-z]', title.copy.truncate(1).upper)
-              author = b.title.split(':').0.trim;
-            %]
-            [%
-            %]
-            <div class="nixos-blog-title">
-              <a [% HTML.attributes(href => b.link) %]>[% title %]</a>
-            </div>
-            <div class="nixos-blog-author-info">
-              <span class="nixos-blog-author">[% HTML.escape(author) %]</span>
-              <span class="muted"> wrote on [% HTML.escape(b.pubDate) %]</span>
-            </div>
-          </div>
-        </li>
-      [% END; END %]
-      <p><a href="https://planet.nixos.org/" class="btn btn-mini">More »</a></p>
-    </ul>
-
-    <h3>Twitter</h3>
+  <div class="span4">
+    <h2>Declarative</h2>
     <p>
-      <a class="twitter-timeline" data-chrome="noheader nofooter transparent" href="https://twitter.com/nixos_org" data-widget-id="433975692670152704">Tweets by @nixos_org</a>
-      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+      Nix makes it <strong>trivial to share development and build
+        environments</strong> for your projects, regardless of what programming
+      languages and tools you’re using.
     </p>
+  </div>
+  <div class="span4">
+    <h2>Reliable</h2>
+    <p>
+      Nix’s ensures that installing or upgrading one package <strong>cannot
+        break other packages</strong>. It allows you to <strong>roll back to
+        previous versions</strong>, and ensures that no package is in an
+      inconsistent state during an upgrade.
+    </p>
+  </div>
+</div>
+
+
+<hr />
 
+<div id="packages-search" class="row-fluid text-center">
+  <div class="span12">
+    <h1>Choose from Thousands of Packages</h1>
+    <p class="text-center lead">
+      The Nix Packages collection (<a href="https://github/NixOS/nixpkgs">nixpkgs</a>) is a set of
+      <strong>over 60 000 packages</strong> for the Nix package manager,
+      released under a permissive <a href="https://spdx.org/licenses/MIT.html">MIT/X11 license</a>.
+    </p>
+    <form>
+      <div class="input-append">
+        <input class="input-xxlarge" placeholder="Search for packages" type="text" />
+        <button class="btn">Search</button>
+      </div>
+      <p class="text-center">or <a href="[% root %]nixos/options.html">search among many NixOS options.</a></p>
+    </form>
   </div>
+</div>
+
+<hr />
+
+<h1 class="text-center">Examples...</h1>
```

Comment:
I had few ideas for this title, but all of them would need to have some more discussion. using "Examples..." is nothing exciting of a title but it makes clear what the next section is. I think this is a good enough title for start and once PR gets merged we can get creative with it.

One of the idea I had is to be a bit more *aggressive* and say: *"Can you tools do this?"*.

Reply:
that's one part i don't like. "Examples" really said nothing. examples for what? but it's ok for now to get the pr merged i guess

What about "Use cases for Nix"?

To get new inspirations, we can look what other projects use as wording for examples... later

Reply:
I will leave it as it is now and lets come back to this in later PRs